### PR TITLE
Set up a "Hello world" project as one app with Webpack

### PR DESCRIPTION
**Pull request by anthonymr**
In this PR I completed all these requirements:

 - Create a new Rails app called 'hello-rails-react'.
      - Use the command `rails new -j webpack my_new_rails_app` as described in the previous lesson's tutorial.
 - Verify that the `jsbundling-rails` gem is added to your Gemfile.
 - Initialize your project with Git.
 - Install needed node dependencies as described in the previous lesson's tutorial.
 - Create a `babel.config.js` in the root folder of your project as described in the previous lesson's tutorial.
 - Update `webpack.config.js` to include the babel loader as described in the previous lesson's tutorial.
 - Make sure that your project has a Postgres database set up. 
 - Create a table for storing your messages and create 5 different greetings. 
 - Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
 - Create a static view that will be the root of your app.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+    <%= javascript_include_tag "application.js", "data-turbo-track": "reload", defer: true %>
```
Comment (brytebee) [inline app/views/layouts/application.html.erb]: image

This link keeps breaking the code as shown in the image above. When commented out the code works but doesn't yield the API. Kindly fix this.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @anthonymr👋,

##

### Highlights💡
- Good RoR project setup and Readme file.✔️ 
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (anthonymr) [reply app/views/layouts/application.html.erb]: Hi @brytebee, thanks for your review.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @anthonymr 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**